Source/Request.swift

```diff
@@ -486,6 +494,18 @@ open class Request {
 
         return self
     }
+
+    // MARK: - Redirects
+
+    /// Sets the redirect handler for the `Request` which will be used if a redirect response is encountered.
+    ///
+    /// - Parameter handler: The redirect handler.
```

Comment:
As with some of my other comments, I've been using the type for most parameter docs, so "The `RedirectHandler`." would be my doc here. 

Also, lining up all of the "The"s to match my previous alignment. Or we can do a documentation pass later.

Reply:
Updated in dd39e600, and yes, we'll catch all this prior to release.